README.md

```diff
@@ -127,6 +127,53 @@ spec:
         hostPath:
           path: /etc/localtime
 ```
+
+For Kubernetes >= 1.9 you need to add selector tag as bellow:
+
+```yaml
+apiVersion: extensions/v1beta1
```

Comment:
Nit:
Since, after kubernetes 1.9, DaemonSet has been move to `apps` group. Bump group version for DaemonSet to `apps/v1` is the recommend configuration.

Reply:
Indeed, sorry